.github/workflows/publish.yml

```diff
@@ -40,11 +40,69 @@ jobs:
       - name: Add package
         id: add_package
         run: |
-          git clone https://github.com/Lombiq/GitHub-Actions.git /tmp/GitHub-Actions
-          cp /tmp/GitHub-Actions/scripts/add-package.sh .
-          bash add-package.sh Microsoft.SourceLink.GitHub
-          rm -Rf /tmp/GitHub-Actions
-          rm add-package.sh
```

Comment:
I found this solution very neat. I didn't really understand, why we can't pull the file from an open repository that we own.

Reply:
See https://github.com/Lombiq/GitHub-Actions/pull/2#discussion_r796065264